README.md

```diff
@@ -44,7 +44,7 @@ examples:
 ```ts
 let localizationService = new LfLocalizationService();
 localizationService.setLanguage('es');
-await localizationService.initResourcesFromUrlAsync('https://cdn.jsdelivr.net/npm/@laserfiche/lf-resource-library@3.0.0/resources/laserfiche-base');
+await localizationService.initResourcesFromUrlAsync('https://cdn.jsdelivr.net/npm/@laserfiche/lf-resource-library@4/resources/laserfiche-base');
```

Comment:
I think the examples also need to be updated to use the culture, and I think there's somewhere where we say en is required.. needs to be en-US